presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -220,4 +232,16 @@ private static void populateDataToHiveTable(String destination, String source, O
         onHive().executeQuery("set hive.enforce.sorting = true");
         onHive().executeQuery(queryStatement);
     }
+
+    private static void populateRowToHiveTable(String destination, String value, Optional<String> partition)
```

Comment:
Use `Collection<Object>` / `Collection<String>` instead of String of comma-separated values. Then the method call is independent from internal SQL syntax and potentially any escaping issues etc.